OSDOCS 8197: Updated oc-mirror v2 section for partially and fully disconnected environment updates (#77247)
openshift/openshift-docs: modules/oc-mirror-workflows-partially-disconnected-v2.adoc

// Module included in the following assemblies:
//
// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc

:_mod-docs-content-type: PROCEDURE
[id="oc-mirror-workflows-partially-disconnected-v2_{context}"]
= Mirroring an image set in a partially disconnected environment

.Mirror To Mirror

You can use this workflow for partially disconnected mirroring, that is, when the environment from which you are executing oc-mirror plugin v2 has both:
- Access to the internet

> - Access to the mirror registry

Author: We have to somewhere mention that it is mirror to mirror.

.Procedure

> . Run the oc mirror command to mirror the images from the specified image set configuration to a specified registry:

Reviewer: ```suggestion
. Mirror the images from the specified image set configuration to a specified registry by running the following command:
```

> . Mirror the images from the specified image set configuration to a specified registry by running the following command:

Reviewer 2: One step procedures should be unnumbered (*)

+
[source,terminal]
----
$ oc mirror -c isc.yaml --workspace file://oc-mirror/mirror1 docker://localhost:6000 --v2

> ----

Reviewer 3: @sherine-k do we need to mention --dest-tls-verify=false  or is that not required ?

.Verification

> . Navigate into the `working-dir` directory that was generated.

Reviewer: ```suggestion
. Navigate to the `working-dir` directory that was generated.
```

> . Navigate into the results directory, for example, `cluster-resources`.

Reviewer: ```suggestion
. Navigate to the results directory, for example, `cluster-resources`.
```

> * Navigate to the `cluster-resources` directory within the `working-dir` directory that was generated at `<file_path>.

Reviewer 2: Missing backtick. Also should qualify what <file_path> is
```suggestion
* Navigate to the `cluster-resources` directory within the `working-dir` directory that was generated in the `<file_path>` directory.
```

. Verify that YAML files are present for the `ImageDigestMirrorSet`, `ImageTagMirrorSet` and `CatalogSource` resources.

.Next steps

> * Configure your cluster to use the resources generated by oc-mirror plugin v2.

Author: I can add this as additional resources in the assembly.

> * Configure your cluster to use the resources generated by oc-mirror plugin v2.

Reviewer: If the cluster configuration has to be done immediately after the image mirroring, we can keep it as the "next step". You can add the xref in the assembly.

> * Configure your cluster to use the resources generated by oc-mirror plugin v2.

Reviewer: This same comment applies to all the instances.
